plugins/woocommerce-docs/tests/src/Manifest/ManifestProcessorTest.php

```diff
@@ -76,4 +76,50 @@ function ( $preempt, $args, $url ) use ( $md_file ) {
 		$this->assertTrue( has_category( $testing_category->term_id, $unit_testing_post->ID ) );
 		$this->assertTrue( has_category( $troubleshoot_category->term_id, $what_went_wrong_post->ID ) );
 	}
+
+	/**
+	 * Test processing a manifest attaches edit url post meta.
+	 */
+	public function test_edit_url() {
```

Comment:
I added this integration test before I did the refactoring, happy to leave it because it has not much cost to include, but also could be convinced to remove it since we unit test the post meta behaviour in PostCreator.